src/type/timer.f90

```diff
@@ -45,99 +47,164 @@ module xtb_type_timer
 
 contains
 
+!--------------------------------------------
+!> To initialize timer
+!--------------------------------------------
 subroutine allocate_timer(self,n,verbose)
+   
    implicit none
+   !> dummy arg list
    class(tb_timer),intent(inout) :: self
+      !! instance of timer
    integer, intent(in)           :: n
+      !! number of timers
    logical, intent(in), optional :: verbose
+      !! if verbose
+
+   !> local vars
    real(wp) :: time_cpu
    real(wp) :: time_wall
+
    call self%deallocate
    if (n < 1) return
-   call timing(time_cpu,time_wall)
+      !! capture negative values
```

Comment:
Both `!!` and `!>` create a docstring for tools like FORD or doxygen, these tools usually get confused if variables and statements are documented this way which are not part of the interface of the procedure.

Reply:
All right, I checked the FORD documentation, and see your point.  
I try to be consistent within the source using only predocmark (`!>`) . 

Did you try to generate xtb-docstring using FORD?

Reply:
Yeah, I tried a while ago, but never deployed because we don't had a good coverage of docstrings so far. Still good to work toward the goal of deploying the whole docs in the future like for [tblite](https://tblite.github.io/tblite/).

Reply:
I rewrote all my comments in a consistent style, so I think this PR is good to go.

It will be helpful to establish some style guide in the best case directly in GitHub, but I am not aware if it is possible. Do you know any means of doing that? 
Another possibility would be just add [here](https://xtb-docs.readthedocs.io/en/latest/contents.html) a **developer guide**.